### PR TITLE
Issue #2275 Exporting functions in testsuite package for Che addon integration tests

### PR DESCRIPTION
**Pull request by agajdosi**

### CODE REVIEW

**Comment by budhrg** [comment]
@agajdosi issue no?

**Comment by agajdosi** [comment]
@budhrg Thanks for noticing, forgot to push latest change with proper commit message and thus the bad PR name.

**Comment by agajdosi** [comment]
@minishift/minishift-dev  Please take a look, only minor changes.

File: test/integration/testsuite/testsuite.go [hunk 1]
```diff
@@ -43,9 +43,9 @@ import (
 )
 
 var (
-	minishift       *Minishift
-	minishiftArgs   string
-	minishiftBinary string
+	MinishiftInstance *Minishift
```
Comment (praveenkumar) [inline test/integration/testsuite/testsuite.go]: :+1:

File: test/integration/testsuite/testsuite.go [hunk 2]
```diff
@@ -434,12 +434,12 @@ func stdoutContainsKey(commandField string, format string, condition string, key
 	return nil
 }
 
-func getLastCommandOutput() CommandOutput {
+func GetLastCommandOutput() CommandOutput {
```
Comment (praveenkumar) [inline test/integration/testsuite/testsuite.go]: :+1:

**Pull request merged**